tox.ini

```diff
@@ -26,8 +25,20 @@ setenv =
 commands=
     python -X dev -X warn_default_encoding -m pytest --durations 25 {posargs}
 
+[testenv:lint]
+description =
+    Run linters.
+extras =
+    lint
+# If you update any of these commands, don't forget to update the equivalent
+# GitHub Workflow step
+commands =
+    ruff . --diff --format github
+    flake8 .
+    isort --check-only --diff .
+    mypy sphinx/
```

Comment:
Don't we also apply mypy to tests?

Reply:
Apparently not: I simply copied these commands from `.github/workflows/lint.yml`

Reply:
Oh good to know. @AA-Turner any reason why we are not applying mypy to tests actually ? (is it perhaps because there would be too many false positive due to monkey patching or other weird constructions?)

Reply:
> (is it perhaps because there would be too many false positive due to monkey patching or other weird constructions?)

Not @AA-Turner but that's usually why I don't run mypy over tests, yes.